test/e2e/allocator_test.go

```diff
@@ -393,3 +389,11 @@ func generateTLSCertPair(t *testing.T, host string) ([]byte, []byte) {
 
 	return pemPubBytes, pemPrivBytes
 }
+
+func validateAllocatorResponse(t *testing.T, resp *pb.AllocationResponse) {
+	t.Helper()
+	assert.Greater(t, len(resp.Ports), 0)
```

Comment:
Could here be assert.Equal(..., 1) or other specific value ?

Reply:
For this test >0 is enough for validation.

Reply:
OK, got it